merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/SimulatedActivityRecord.java

```diff
@@ -14,7 +14,7 @@
     Map<String, SerializedValue> parameters,
     Instant start,
     Duration duration,
-    Long parentId,
-    List<Long> childIds,
+    ActivityInstanceId parentId,
```

Comment:
Hmm... I thought the reason for SimulatedActivityRecord to exist was that it used postgres ids, which were distinct from merlin ids....

Does it still make sense for it to exist as a separate concept from SimulatedActivity?

Reply:
Without looking too deep, I lean toward agreeing with Matt on this -- the fact that it's a `long` is important to the Postgres binding layer, and obfuscating it doesn't help in this component. Any e*long*ation of IDs should probably happen wherever the `SimulatedActivityRecord` is converted into something else.